src/tabixIndexedFile.ts

```diff
@@ -231,11 +216,9 @@ export default class TabixIndexedFile {
         blockStart += line.length + 1
 
         // yield if we have emitted beyond the yield limit
-        linesSinceLastYield += 1
-        if (linesSinceLastYield >= this.yieldLimit) {
+        if (last - perf.now() > 10000) {
+          last = perf.now()
           await timeout(1)
-          checkAbortSignal(signal)
```

Comment:
probably want to add that abort signal check back in, right?

Reply:
added back